core/src/components/fab/test/translucent/fab.e2e.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test('should not have visual regressions', async ({ page, skip }) => {
+  skip.rtl();
+  skip.mode('md', 'Translucency is only available on ios mode');
+  await page.goto(`/src/components/fab/test/translucent`);
+
+  const fab = page.locator('#fab5');
+  await fab.click();
+  await page.waitForChanges();
+
+  // fab.screenshot doesn't work since the bounding box is just the middle button
+  await page.setViewportSize({
```

Comment:
Similar feedback here - can we add a note why we set this explicit size, e.g.:
> // We assign a viewport size that is cropped, but contains the visible FABs to avoid capturing extra whitespace in the screenshot.